src/components/FloatingActionButton.js

```diff
@@ -23,7 +23,8 @@ const FloatingActionButton = ({ label, iconShape, onClick, bottom = 30, right =
       onClick: () => {
         onClick()
         setHover(false)
-      }
+      },
+      ...props
```

Comment:
This isn't right, I think. The ref you're passing here is creating a warning. I believe you should be able to find the right pattern elsewhere in the codebase.

Reply:
In the FloatingActionButton? I'm not sure I understand which ref you're referring to. The only thing I changed in this file was to pass through additional optional properties to the button. That said, I don't recall now whether I actually ended up needing the extra props here. I'll double-check and remove them if I can.

Reply:
It's used to pass a ref on line 683 of Upload.js. I'd say leave that out until we can get it right. We can do a usability pass later.

Reply:
(the ref is used to focus a button)